test/tests/functional/pbs_equiv_classes.py

```diff
@@ -1773,145 +1753,124 @@ def test_multiple_equivalence_class_preemption(self):
         self.server.expect(JOB, {'job_state': 'R'}, id=jid5)
         self.server.expect(JOB, {'job_state': 'R'}, id=jid6)
 
-        """Test only 3 equivalence class are present, i.e one
-         equivalence class for jid1 and jid3,1 equi class for jid2
-          and jid4 and 1 equi class for jid5 and jid6 """
+        #Test only 3 equivalence class are present, i.e 1
+        #equivalence class for jid1 and jid3,1 equivalence class for jid2
+        #and jid4 and 1 equivalence class for jid5 and jid6
 
         self.scheduler.log_match("Number of job equivalence classes: 3",
                                  max_attempts=10, starttime=self.t)
```

Comment:
you will see 3 equivalence classes only if jid1 and jid3 were run on same vnode. There might be cases when jid3 could be picked up by a different node than you see jid1 and jid3 in different equivalence classes. Also looking at the objective of this test you don't need 2 vnodes. Either you create one vnode with 2 ncpus or set the node priority on each vnode so that node1 is always considered first.

Reply:
changed this test to include 1 node